lib/hardware_services/network_drivers/hil.yml

```diff
@@ -0,0 +1,2 @@
+url: http://127.0.0.1:5000    # the address of HIL server
```

Comment:
@vsemp I think this file is not necessary as we now have this as a parameter in the conf/quads.yml file, as we discussed.